addons/website/static/src/js/content/website_root.js

```diff
@@ -207,6 +245,20 @@ var WebsiteRoot = publicRootData.PublicRoot.extend({
     _onModalShown: function (ev) {
         $(ev.target).addClass('modal_shown');
     },
+    /**
+     * @override
+     */
+    _onKeyDown(ev) {
+        if (!odoo.session_info.user_id) {
```

Comment:
We try not to use the `odoo` object except in low-level stuff. You can `require` `web.session` then use `session.user_id`